Cargo.toml

```diff
@@ -3,5 +3,6 @@ members = [
     "exonum",
     "sandbox",
     "testkit",
+    "services/exonum-time"
```

Comment:
I wonder if it makes sense to call the directory `exonum-time`; I'd prefer naming it simply `time`.

Reply:
Funny thing: we discussed that with @SerhiiOryshych, but decided to use full crate name. On the other hand `teskit`'s directory isn't named `exonum-teskit`.